public/assets/javascript/saved.js

```diff
@@ -0,0 +1,94 @@
+$(document).ready(function() {
+	function findSavedArticles() {
```

Comment:
same here, you dont want to build the function when the page is ready, if possible you should do it before the page is ready so that you can just call it.